### PR TITLE
Header html

### PR DESCRIPTION
**Pull request by Ibanez92**

### CODE REVIEW

**Comment by Ibanez92** [comment]
looks good

File: assets/style.css [hunk 1]
```diff
@@ -7,12 +7,14 @@ html {
 font-size: larger;
 margin-bottom: 40px;
 margin-top: 10px;}
+
```
Comment (Ibanez92) [inline assets/style.css]: good

File: assets/style.css [hunk 2]
```diff
@@ -59,17 +64,7 @@ header {
...
-  padding: 10px;
-  box-shadow: 0 0 10px 2px;
-    align-content: center;
-    margin: 50px;
-    margin-left: 200px;
-    margin-right: 200px;
-  padding-left: 20px;
-  padding-right: 20px;
-}
+
```
Comment (Ibanez92) [inline assets/style.css:67]: good

**Review by Ibanez92 — COMMENTED** [review]
example

**Review by Crowe828 — APPROVED** [review]

**Pull request merged**